src/app/loans/loans-account-stepper/loans-account-preview-step/loans-account-preview-step.component.ts

```diff
@@ -25,8 +25,19 @@ export class LoansAccountPreviewStepComponent implements OnInit {
   /** Overdue Charges Displayed Columns */
   overdueChargesDisplayedColumns: string[] = ['name', 'type', 'amount', 'collectedon'];
 
+  isLoanSubmitButtonDisabled = false
+
   constructor() { }
 
   ngOnInit() { }
 
+  submitAndDisbleLoanSubmitButton(){
+    this.isLoanSubmitButtonDisabled = true;
+    setTimeout(() => {
+      this.isLoanSubmitButtonDisabled = false;
```

Comment:
Can we add value as environment variable, they should also be added to the env.js and eve.template.js so we can update them from docker.